libvast/test/pipeline.cpp

```diff
@@ -253,10 +247,9 @@ TEST(taste 10 with overlap) {
 }
 
 TEST(head and taste fail with negative limit) {
-  const auto* vastql
-    = vast::plugins::find<vast::query_language_plugin>("vastql");
-  REQUIRE_ERROR(vastql->make_query("head -1"));
-  REQUIRE_ERROR(vastql->make_query("taste -5"));
+  const auto* vast = vast::plugins::find<vast::language_plugin>("VAST");
```

Comment:
Just curious, as it was there before: why do we need `vast::` qualification?

Reply:
Because `pipeline.cpp` did not declare the `vast` namespace. I added a `using namespace` declaration in https://github.com/tenzir/vast/pull/2959/commits/74a21c7b16f6d025f40226be5bd32657cf792cd3

Reply:
Replaced "VAST Language" with "VAST language" in https://github.com/tenzir/vast/pull/2959/commits/4b289bb23fce3c786bad9b58305f8910e4d0182a